spec/features/merchants/invoices/show_spec.rb

```diff
@@ -1,6 +1,22 @@
 require 'rails_helper'
 
 RSpec.describe 'merchant invoices show page' do
+
```

Comment:
Would it be worth it to create some content not to have to make a more robust test suite?